modules/controls.js

```diff
@@ -89,6 +89,9 @@ displaySystem.registerModule({
                 var win = window.open(config.url,'fllDisplayControlWindow','resize=yes,width=800,height=550');
             } else {
                 var win = window.open('','fllDisplayControlWindow','resize=yes,width=800,height=550');
+                if (win.document.body.innerHTML) {
+                    return;
+                }
                 win.document.write(html);
```

Comment:
Unrelated to this PR, but why would we only do stuff with `win` here, and not in the other case? (`if (config.url)`)

Reply:
The url case is the case where you have some completely custom control window, with all controls rendered etc. In that case, we don't bother rendering stuff. I'll add a comment as well